### DISCUSSION TITLE
Add toggle enabled/disabled on side the opened right modal entry.

### DISCUSSION THREAD

**Discussion by vitalijalbu** [craftcms/cms#10658] [Ideas] [answered]
<img width="1597" alt="Schermata 2022-02-28 alle 14 09 48" src="https://user-images.githubusercontent.com/40239902/155989080-4c095394-94ea-4be7-a343-2514d2016514.png">

Would be better to add also here the enable/disable toggle button of the entry.

**Comment by brandonkelly** [comment, marked as answer]
Already added for Craft 4! (#10467)
